### ISSUE TITLE
[BUG]:When adding a workspace, an infinite spinner is observed

### ISSUE THREAD

**Issue by willbertSmillbert** [microsoft/playwright-testing-service#92]
**To Reproduce**
Go to https://playwright.microsoft.com/create-workspace
Attempt to add a workspace (none currently exist) - fill out required field

Observe infinite spinner, no workspace gets created

**Expected behavior**
Creates the workspace

**Screenshots**
![image](https://github.com/microsoft/playwright-testing-service/assets/69780913/e9bdcfd6-de52-409e-abe0-30ec929b5c09)
![image](https://github.com/microsoft/playwright-testing-service/assets/69780913/a3220995-768e-48d1-b013-2bf788fd872e)
![image](https://github.com/microsoft/playwright-testing-service/assets/69780913/5da93713-12a4-411c-8b14-fc325757a498)

**Setup information (please complete the following information):**
 - Azure Subscription Id: 15451e06-b6e5-4b64-ada2-1bf419245949
 - Time  the issue was experienced: 2024-02-01T01:13:59.9374693Z
 
**Additional context**
There must be something special about my azure subscription, manually making a workspace through the portal (with a region supported by playwright) also results in the workspace not showing up in the workspace list in playwright - unsure if a playwright workspace is seperate from the general workspaces in azure portal.

Let me know what else you need. As you can see I cant get very far in attempting to try out this new service..

AB#1949833

**Comment by willbertSmillbert** [comment]
I got another user to make the playwright workspace through Azure portal - this worked fine.

I am no longer blocked - but still confident that the provisioning stage from the front end for workspaces is still boned (for me atlteast)

**Comment by puagarwa** [comment]
Hi @willbertSmillbert thanks for reporting the issue and glad that you are unblocked for now. We will check the issue in our telemetry and respond back on why this happened.

**Comment by mjmadhu** [comment]
@willbertSmillbert Thanks for reporting the issue. We looked into our logs and it seems to be some transient issue causing delays in subscription registration with playwright service and this should not happen again.
You can please re-try and let us know if you're still hitting any roadblock in this operation.
_P.S. - It'd be nice to have some network logs and correlation id header from the request/response from the network calls to help debug faster._

Also, w.r.t. to your ask -
_There must be something special about my azure subscription, manually making a workspace through the portal (with a region supported by playwright) also results in the workspace not showing up in the workspace list in playwright - unsure if a playwright workspace is seperate from the general workspaces in azure portal._

You should be able to create resource under Playwright Testing from Azure Portal in case that's not what you created earlier. Please feel free to report here in case you face any issues there as well.

![image](https://github.com/microsoft/playwright-testing-service/assets/71822730/a43edc55-4db5-464f-bcf2-1d366f4faaa6)

**Comment by willbertSmillbert** [comment]
@mjmadhu Hey, Thanks for the quick reply. Yea - we got unblocked earlier by doing it through the portal - I just confirmed this creation flow also works with my user through the Azure Portal.

Im still experiencing issues with the creation of the workspace through the playwright service.

See below the calls that my client is making when attempting to create the workspace

![image](https://github.com/microsoft/playwright-testing-service/assets/69780913/a7fc249c-bce1-4294-b79d-639d95d21de7)


X-Ms-Client-Request-Id: 0443d048-aee0-4d9e-94a0-b8a17529c3d1 (for one of them)

**Comment by mjmadhu** [comment]
Thanks so much for quick confirmation @willbertSmillbert and useful tracking information. We'll check this and get back to you.

**Comment by mjmadhu** [comment]
@willbertSmillbert Thanks for your patience and apologies for the inconvenience. We'd identified the issue and worked on the fix. Please give it a retry and let us know if you're able to proceed with the operation from Playwright Dashboard or still facing any issue (tracking details would be useful as earlier).

**Comment by willbertSmillbert** [comment]
Tested
Working as expected
